### DISCUSSION TITLE
A big THANK YOU!

### DISCUSSION THREAD

**Discussion by tobehn** [thomasnordquist/MQTT-Explorer#791] [Show and tell]
I really appreciate the ongoing development of this awesome piece of software.
The sparkplug B decoding made me really happy today.

🙏

**Comment by thomasnordquist** [comment]
@bj00rn did A LOT to get this project living again

**Reply by ilgrank** [reply]
We owe you and @bj00rn a great deal, thanks for coming back to MQTT-Explorer!

**Comment by JemRF** [comment]
Is Sparkplug on the current release .4 beta 6 ?
I do not see an option.

**Reply by tobehn** [reply]
It has been deactivated due to some issues with the sparkplugB decoding -> #792.
This seems to be already fixed and merged with #793. Though there haven't been further releases since beta 6.
See https://github.com/thomasnordquist/MQTT-Explorer/releases/tag/v0.4.0-beta.6

To test Sparkplug B implementation you could install beta 4 - i had no problems with deconding in that version

**Comment by jjauzion** [comment]
Many thx for this awesome project indeed :)
Any idea when the Sparkplug will be released ? I'm using the [beta4](https://github.com/thomasnordquist/MQTT-Explorer/releases/tag/0.4.0-beta4) for the moment but would love to have it in a stable version
